catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/DeleteLookBrickTest.java

```diff
@@ -95,10 +96,11 @@ public void testPaintLookWithoutDelete() {
 
 	@Category({Cat.AppUi.class, Level.Smoke.class})
 	@Test
-	public void testPaintAndDeleteLook() {
+	public void testPaintAndDeleteLook() throws InterruptedException {
 		script.addBrick(new DeleteLookBrick());
 
 		onView(withId(R.id.button_play)).perform(click());
+		Thread.sleep(100);
 		onView(withId(R.id.pocketpaint_drawing_surface_view)).perform(click());
 		pressBack();
 
```

Comment:
```suggestion
		Thread.sleep(100);
		onView(withId(R.id.pocketpaint_drawing_surface_view)).perform(click());
		pressBack();
		Thread.sleep(100);
```
Just to make sure that the DeleteLookBrick from line 100 has really been executed, since it is placed _after_ the PaintNewLookBrick from line 74 in the script, and thus the assert would have to wait for its execution.